src/routes/(app)/news/LikersList.svelte

```diff
@@ -2,23 +2,33 @@
   import { getFullName } from "$lib/utils/client/member";
   import type { Member } from "@prisma/client";
   import AuthorSignature from "$lib/components/AuthorSignature.svelte";
+  import * as m from "$paraglide/messages";
 
   let modal: HTMLDialogElement;
   export let likers: Member[];
   $: likersText =
-    likers.length > 2
-      ? `${getFullName(likers[0]!)}, ${getFullName(likers[1]!)} och ${
-          likers.length - 2
-        } andra`
-      : `${likers.map((m) => getFullName(m)).join(" och ")}`;
+    likers.length > 0
+      ? likers.length === 1
+        ? getFullName(likers[0]!)
+        : likers.length > 2
+          ? m.news_threeOrMore({
+              name1: getFullName(likers[0]!),
+              name2: getFullName(likers[1]!),
+              others: likers.length - 2,
+            })
+          : m.news_two({
+              name1: getFullName(likers[0]!),
+              name2: getFullName(likers[1]!),
+            })
+      : "";
```

Comment:
See my comment on #379 for a similar thing, nested ternaries are bad and this gets quite complex and hard to read.